packages/fxa-shared/metrics/glean/fxa-backend-metrics.yaml

```diff
@@ -986,3 +986,81 @@ two_factor_auth_setup:
     expires: never
     data_sensitivity:
       - interaction
+
+inactive_account_deletion:
+  status_checked:
+    type: event
+    description: |
+      The active status of an account was checked.
+    lifetime: ping
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-9587
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+    expires: never
+    data_sensitivity:
+      - technical
+
+  first_email_task_request:
+    type: event
+    description: |
+      An attempt to insert the first email task into the cloud task queue.
+    lifetime: ping
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-9587
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+    expires: never
+    data_sensitivity:
+      - technical
+
+  first_email_task_enqueued:
+    type: event
+    description: |
+      The first email task was successfully added to the cloud task queue.
+    lifetime: ping
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-9587
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+    expires: never
+    data_sensitivity:
+      - technical
+
+  first_email_task_rejected:
+    type: event
+    description: |
+      The first email task was not added the cloud task queue.
+    lifetime: ping
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-9587
```

Comment:
I think the jira ticket in reference to all the glean metrics needs to be updated.